src/app/pages/send-crypto-asset/send-crypto-asset-form/components/use-font-resizer.ts

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useOnMount } from '@app/common/hooks/use-on-mount';
+
+export const maxWidth = 344;
+const maxFontSize = 48;
+const minFontSize = 14;
+
+export function useFontResizer() {
+  const [inputFontSize, setInputFontSize] = useState(maxFontSize + 'px');
+
+  useOnMount(() => {
+    const input = document.getElementById('input');
```

Comment:
Since this gets an element globally on the page, perhaps we should use a more specific id name?

Maybe to make it a bit more specific the name of the input can be passed in as an argument?